internal/ksql/command_cluster_create.go

```diff
@@ -8,11 +8,16 @@ import (
 
 	pcmd "github.com/confluentinc/cli/v4/pkg/cmd"
 	"github.com/confluentinc/cli/v4/pkg/examples"
+	"github.com/confluentinc/cli/v4/pkg/featureflags"
 	"github.com/confluentinc/cli/v4/pkg/kafka"
 	"github.com/confluentinc/cli/v4/pkg/log"
 	"github.com/confluentinc/cli/v4/pkg/output"
 )
 
+const (
+	ksqlMinCSUDeadline = "January 15, 2026"
```

Comment:
is this to be hardcoded or is it better to read it from a LD flag?

Reply:
I agree, we shouldn't hard code this.